Spigot-Server-Patches/0348-Fix-sounds-when-item-frames-are-modified-MC-123450.patch

```diff
@@ -18,16 +18,3 @@ index 584f64946821092afab57b9409bc249403bc16e7..43152a6c70c9433d627a58051101530d
              this.playSound(SoundEffects.ENTITY_ITEM_FRAME_ADD_ITEM, 1.0F, 1.0F);
          }
  
-diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftItemFrame.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftItemFrame.java
-index 86bb26cd1d327056a64926ed6fc53142ad3756c2..cbdbb89ec28f9b6c3a0eb31be94c440254c6e266 100644
---- a/src/main/java/org/bukkit/craftbukkit/entity/CraftItemFrame.java
-+++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftItemFrame.java
-@@ -50,7 +50,7 @@ public class CraftItemFrame extends CraftHanging implements ItemFrame {
-         old.die();
- 
-         EntityItemFrame frame = new EntityItemFrame(world, position, direction);
--        frame.setItem(item);
-+        frame.setItem(item, true, false); // Paper - fix itemframe sound
```

Comment:
just to double check, this was dropped intentionally, right?

Reply:
Believe that was fixed upstream:
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/commits/32e4086604f6df169869172175a194c41422de38#src/main/java/org/bukkit/craftbukkit/entity/CraftItemFrame.java